lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -76,6 +77,16 @@ static FIRRTLType getFIRRTLType(Type type) {
         unsigned width = indexType.kInternalStorageBitWidth;
         return UIntType::get(context, width);
       })
+      .Case<TupleType>([&](TupleType tupleType) -> FIRRTLType {
+        using BundleElement = BundleType::BundleElement;
+        llvm::SmallVector<BundleElement> elements;
+        unsigned fieldCnt = 0;
+        for (auto elementType : tupleType.getTypes())
```

Comment:
`for (auto it : llvm::enumerate(tupleType.getTypes()))`

Reply:
Didn't know that this exists, thanks.